### ISSUE TITLE
Crash on iOS when navigating too quickly with a11y means

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#13701]
For example, If you change from one screen to another and then quickly try to focus on something on the new screen, the app crashes (no useful exception is given).

**Comment by yjbanov** [comment]
This is fixed in the engine. No code changes are necessary on the framework side. The fix should be automatic next time the engine is updated.
